src/main/java/frc/robot/commands/HoodHomeCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.ShooterSubsystem;
+
+public class HoodHomeCommand extends CommandBase {
+    private ShooterSubsystem shoot;
+
+    public HoodHomeCommand(ShooterSubsystem shoot) {
+        this.shoot=shoot;
+    }
+
+    // Called just before this Command runs the first time
+    @Override
+    public void initialize() {
+        SmartDashboard.putBoolean("Home called", true);
```

Comment:
We need to be consistent with how we do one-shot commands. See above, I recommend using a base class